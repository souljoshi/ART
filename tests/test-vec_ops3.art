int main()
{
	vec v1;
	vec v2;
	double f1;
	double f2;

	f1 = 2.0;
	f2 = 5.5;

    v1 = <1.0, 55.1> * f1;
    v2 = f2 * v1;
    v1 = v1 + v2;

    printf(v1[0]);
    printf(v1[1]);
    printf(v2[0]);
    printf(v2[1]);

}