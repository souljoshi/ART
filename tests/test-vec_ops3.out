13.000000
716.300000
11.000000
606.100000
